packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -1,31 +1,26 @@
 import {h} from '@stencil/core';
+import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
 import ClearIcon from 'coveo-styleguide/resources/icons/svg/clear.svg';
-import {Combobox} from '../../../utils/combobox';
-import {Bindings, I18nState} from '../../../utils/initialization-utils';
-import {
-  CategoryFacet,
-  Facet,
-  SpecificFacetSearchResult,
-  CategoryFacetSearchResult,
-} from '@coveo/headless';
+import {Combobox, ComboboxStrings} from '../../../utils/combobox';
+import {Facet, CategoryFacet, CategoryFacetSearchResult} from '@coveo/headless';
 import {randomID} from '../../../utils/utils';
-import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
 import {sanitize} from '../../../utils/xss-utils';
 import {regexEncode} from '../../../utils/string-utils';
 
-type FacetSearchResult = SpecificFacetSearchResult &
-  Partial<Pick<CategoryFacetSearchResult, 'path'>>;
+type FacetSearchResult = CategoryFacetSearchResult;
```

Comment:
I can remove the alias, this is mainly for Typescript validation reason, since CategoryFacetSearchResult contain "path" keeping `CategoryFacetSearchResult | SpecificFacetSearchResult` was giving TS errors